packages/contracts-bedrock/contracts/EAS/SchemaRegistry.sol

```diff
@@ -0,0 +1,95 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.19;
+
+import { Initializable } from "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+
+import { Semver } from "../universal/Semver.sol";
+
+import { ISchemaResolver } from "./resolver/ISchemaResolver.sol";
+
+import { EMPTY_UID, MAX_GAP } from "./Common.sol";
+import { ISchemaRegistry, SchemaRecord } from "./ISchemaRegistry.sol";
+
+/**
+ * @title The global schema registry.
+ */
+contract SchemaRegistry is ISchemaRegistry, Semver, Initializable {
+    error AlreadyExists();
+
+    // The global mapping between schema records and their IDs.
+    mapping(bytes32 uid => SchemaRecord schemaRecord) private _registry;
+
+    // Upgrade forward-compatibility storage gap
+    uint256[MAX_GAP - 1] private __gap;
+
+    /**
+     * @dev Creates a new SchemaRegistry instance.
+     */
+    constructor() Semver(1, 0, 0) {}
```

Comment:
Confused as to why this is `Initializable`  when no storage is set in the constructor

Reply:
See the previous comment re: EAS (i.e., it's just a convention), but in any case I'll remove this.

Reply:
I've removed the initializers. Please take a look.